### DISCUSSION TITLE
Graduated but still on Github Student Developer Pack

### DISCUSSION THREAD

**Discussion by hilmandayo** [community/community#37351] [GitHub Education]
Hey guys. I applied for the Student Developer Pack and graduated from the University right now.
However, it seems like I am still subscribed to the Developer Pack. Do I need to unsubscribed to it or I can just use it until it expired?

**Comment by samus-aran** [comment]
Hi @hilmandayo 👋 Congratulations on graduating 🎉 
Enjoy the benefits until they expire and do amazing stuff with them 👍
